subscriptions/subscriptions-api/src/main/java/com/duckduckgo/subscriptions/api/SubscriptionScreens.kt

```diff
@@ -22,4 +22,5 @@ sealed class SubscriptionScreens {
     data object SubscriptionsSettingsScreenWithEmptyParams : ActivityParams
     data object SubscriptionScreenNoParams : ActivityParams
     data class RestoreSubscriptionScreenWithParams(val isOriginWeb: Boolean = true) : ActivityParams
+    data class SubscriptionPurchaseWithOrigin(val origin: String? = null) : ActivityParams
```

Comment:
nit: If the `origin` param is optional here, what is the difference between using `SubscriptionPurchaseWithOrigin()` vs `SubscriptionScreenNoParams`? If there is no difference, then we could clean this public API up by merging those two into a single class `data class SubscriptionPurchase(val origin: String? = null) : ActivityParams`